go.mod

```diff
@@ -97,7 +97,7 @@ require (
 	github.com/felixge/httpsnoop v1.0.4 // indirect
 	github.com/fxamacker/cbor/v2 v2.7.0 // indirect
 	github.com/gabriel-vasile/mimetype v1.4.2 // indirect
-	github.com/go-jose/go-jose/v4 v4.0.1 // indirect
+	github.com/go-jose/go-jose/v4 v4.0.5 // indirect
```

Comment:
I don't think updating indirect dependency will do the job here, since it's coming from one of the direct dependencies and we should fix them instead

Reply:
updating indirect ones might be a temporary solution, but may come up again if the parent dependency updates it